code/src/components/GameContent.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { actionsGame } from 'reducers/game';
+// import { Map } from './Map.js'
+
+export const GameContent = () => {
+  const { actions, coordinates } = useSelector((store) => store.game.position)
+  // const { description, actions, coordinates } = useSelector((store) => store.game.position)
+  const dispatch = useDispatch()
+  const username = useSelector((store) => store.game.username)
+
+  const onDirectionBtnClick = (direction) => {
+    dispatch(actionsGame(direction))
+  }
+
+  const onRestartBtnClick = () => {
+    return (
+      window.location.reload()
+    )
+  }
+
+  return (
+    <div className="name-content-container">
+      <player>
+        <h2>{username}</h2>
+        <button type="button" onClick={onRestartBtnClick}>Restart game
+        </button>
+      </player>
+      <div className="game-container">
+        <div className="navigation-container">
+          <div className="actions-container">
+            {actions.map((singleAction) => (
```

Comment:
I think `singleAction` should be a different colour if it is working, so perhaps something is wrong with `singleAction` - and that might be why your button click doesn’t load the next move…. 
Could it be something to do with line 7?